code/ShareCare.php

```diff
@@ -237,6 +237,20 @@ public function getDefaultOGTitle()
     }
 
     /**
+     * Get the main OG image (even if it's an array of images)
+     *
+     * @return string
+     */
+    public function getMainOGImage()
```

Comment:
Could you change this to `getFirstOGImage`? I feel like that's more descriptive and less likely to confuse me later.